docs/en/reference/working-with-associations.rst

```diff
@@ -500,6 +500,19 @@ and the "remove" operation.
         //...
     }
 
+Since ``cascade: persist`` is configured for the ``User#commentsAuthored``
+association, you can now create a user and persist his comment as follows:
```

Comment:
I'm not 100% sure what you mean. However:
* You can't call a person "it"! I followed the convention on this page, referring to a user as "he".
* The example is about just one comment, not many.

If you mean something else, let me know :-)